pygridtools/iotools.py

```diff
@@ -14,6 +14,7 @@
     ipywidgets = None
 
 from pygridgen.tests.utils import requires
+from pygridgen import grid as pyggrid
```

Comment:
let's just `import pygridgen as pgg` to be consistent with the docs

Reply:
sounds good